st2common/st2common/models/db/execution.py

```diff
@@ -71,9 +71,7 @@ class ActionExecutionDB(stormbase.StormFoundationDB):
             {'fields': ['end_timestamp']},
             {'fields': ['status']},
             {'fields': ['parent']},
-            {'fields': ['-start_timestamp', 'action']},
-            {'fields': ['-start_timestamp', 'parent']},
-            {'fields': ['-start_timestamp', 'action', 'parent']}
```

Comment:
I removed them and didn't see any difference in the different `st2 execution list` queries.